bevy_ios_safearea/Cargo.toml

```diff
@@ -10,11 +10,15 @@ keywords = ["bevy", "gamedev", "mobile", "ios", "swift"]
 description = "Bevy plugin to query device safe area insets"
 
 [package.metadata.docs.rs]
-default-target = "x86_64-apple-ios"
+default-target = "x86_64-apple-darwin"
```

Comment:
this does not make sense to me as our features configs are using ios and not macos. if this is to test if it fixes docs.rs lets do that separately

Reply:
Let me do a 0.1.5 then xD